build/tools/build_helper.fd

```diff
@@ -415,7 +415,13 @@ install_opencoord.org_freediameter() {
     if ! patch -R -p1 -s -f --dry-run <  $OPENAIRCN_DIR/build/patches/0001-opencoord.org.freeDiameter.patch; then
           patch -p1 <  $OPENAIRCN_DIR/build/patches/0001-opencoord.org.freeDiameter.patch
     else
-      echo_info "Detected freeDiameter $OPENAIRCN_DIR/build/patches/0001-opencoord.org.freeDiameter.patch alredy applied"
+      echo_info "Detected freeDiameter $OPENAIRCN_DIR/build/patches/0001-opencoord.org.freeDiameter.patch already applied"
```

Comment:
@lionelgo 

This new patch is also applied on the RHEL8 builds? Just asking

Reply:
Yes, this for all distribs: 
in init_submodules()
....
  build_freeDiameter
  ret=$?;[[ $ret -ne 0 ]] && return $ret
  build_c_ares
  ret=$?;[[ $ret -ne 0 ]] && return $ret
  build_cpp_driver
  ret=$?;[[ $ret -ne 0 ]] && return $ret

called by check_install_hss_software()
called by build_hss_rel14 script